aerospike/pyproject.toml

```diff
@@ -9,7 +9,6 @@ build-backend = "hatchling.build"
 name = "datadog-aerospike"
 description = "The Aerospike check"
 readme = "README.md"
-license = {text = "BSD-3-Clause"}
```

Comment:
Wondering why this is being changed/which dependency this is for? I remember this needed to be a dictionary on newer versions of `setuptools` and [was failing CI](https://github.com/DataDog/integrations-core/pull/11937).

Reply:
Also if this is getting changed, it doesn't seem to be applied on all integrations? For example, ArangoDB has the [same format](https://github.com/DataDog/integrations-core/blob/master/arangodb/pyproject.toml#L11), but it wasn't included in the PR

Reply:
I am afraid this change is result of running `ddev dep updates --sync`

Reply:
I saw this earlier this week and was talking to ofek about it. This reformatting is due to the toml library we use- only the integrations' `pyproject.toml`s that need to be regenerated (since they had a dependency bump) are getting updated to this format- I guess this is the default way to display a dictionary in toml? I made a card for someone to reformat all integrations so it's consistent